doc/admin-guide/files/records.config.en.rst

```diff
@@ -643,6 +644,8 @@ tr-out
 tr-pass
    Transparent pass through. This option is useful only for inbound transparent proxy ports. If the parsing of the expected HTTP header fails, then the transaction is switched to a blind tunnel instead of generating an error response to the client. It effectively enables :ts:cv:`proxy.config.http.use_client_target_addr` for the transaction as there is no other place to obtain the origin server address.
 
+no-throttle
```

Comment:
The idea is you want to have a port open for critical communications like health checks and other cross ATS communication.  I updated the documentation to at least put in a basic description.  I can expand on that as needed.